openpype/hosts/tvpaint/plugins/load/load_workfile.py

```diff
@@ -0,0 +1,77 @@
+import getpass
+import os
+
+from avalon.tvpaint import lib, pipeline
+from avalon import api, io
+
+from openpype import Anatomy
+
+
+class LoadWorkfile(pipeline.Loader):
+    """Load workfile."""
+
+    families = ["workfile"]
+    representations = ["tvpp"]
+
+    label = "Load Workfile"
+
+    def load(self, context, name, namespace, options):
+        filepath = self.fname.replace("\\", "/")
+
+        if not os.path.exists(filepath):
+            raise FileExistsError(
+                "The loaded file does not exist. Try downloading it first."
+            )
+
+        george_script = "tv_LoadProject '\"'\"{}\"'\"'".format(
+            filepath
+        )
+        lib.execute_george_through_file(george_script)
+
+        # Save workfile.
+        host = api.registered_host()
+
+        project = io.find_one({
+            "type": "project"
+        })
+        session = api.Session
+        data = {
+            "project": {
+                "name": project["name"],
+                "code": project["data"].get("code")
+            },
+            "asset": session["AVALON_ASSET"],
+            "task": session["AVALON_TASK"],
+            "version": 1,
+            "user": getpass.getuser()
+        }
+        anatomy = Anatomy(project["name"])
+        template = anatomy.templates["work"]["file"]
```

Comment:
It is possible that new context will require different work template than `"work"`. It will be required to get template name using `openpype.lib.get_workfile_template_key_from_context`, then recalculate workdir and then you can get last workfile version.

Reply:
Done.
I would say that maybe we should throw an error or warning when using `host.work_root(session)` cause that is not context aware?
Dont think we can make it context aware either cause you would be cross references between Openpype and Avalon.

Reply:
> Dont think we can make it context aware either cause you would be cross references between Openpype and Avalon.

Damn, I didn't think about it but you're right.